Specs/DataSources/KmlDataSourceSpec.js

```diff
@@ -1555,6 +1555,32 @@ defineSuite([
         });
     });
 
+    it('IconStyle: Sets billboard image with query', function() {
+        var kml = '<?xml version="1.0" encoding="UTF-8"?>\
+          <Placemark>\
+              <Style>\
+                  <IconStyle>\
+                      <Icon>\
+                          <href>image.png</href>\
+                      </Icon>\
+                  </IconStyle>\
+              </Style>\
+          </Placemark>';
+        debugger;
```

Comment:
You forgot to take this out (always try and remember to `npm run eslint` before each commit/push)